src/core/src/service_interfaces/StatusHandler.py

```diff
@@ -65,6 +70,13 @@ def __init__(self, env_layer, execution_config, composite_logger, telemetry_writ
 
         self.__current_operation = None
 
+        # Update patch metadata summary in status for auto patching installation requests, to be reported to health store
+        if execution_config.patch_rollout_id is not None and execution_config.operation.lower() == Constants.INSTALLATION.lower():
+            if self.__installation_reboot_status != Constants.RebootStatus.STARTED:
+                self.set_patch_metadata_for_health_store_substatus_json(report_to_health_store=True, wait_after_update=True)
```

Comment:
Please include an else condition and log why nothing is being written. This is to decipher the flow of the code looking only at logs. Applies in general everywhere.